neoforge/build.gradle

```diff
@@ -0,0 +1,114 @@
+plugins {
+    id 'idea'
+    id 'maven-publish'
+    id 'net.neoforged.gradle'
+    id 'org.spongepowered.mixin'
+}
+
+base {
+    archivesName = "${mod_name}-neoforged-${minecraft_version}"
```

Comment:
One thing I do have with this is the `neoforged` here, I would like to keep it matching the existing `neo` like so here https://github.com/AzureDoom/AzureLib/blob/1.20/NeoForge/gradle.properties#L15 to avoid anyone updating having issues with it not finding the correct jar. 

I do agree it is a good change though and once 1.20.2 comes out, it's worth the name change there.